Sources/SwiftFCSDKiOS/SwiftFCSDKiOS.docc/VirtualBackground.md

```diff
@@ -0,0 +1,67 @@
+# VirtualBackground
+
+FCSDKiOS offers virtual background support. In order to use this feature, you will need to be able to provide a UIImage to FCSDKiOS. The SDK consumer is responsible for setting up any UI that stores and presents images for selection. If the Image is too large, FCSDK will size the image to an acceptable size, but in practice it is encouraged for the SDK consumer to handle this to prevent any undefined behavior. Typically you will want to take the resoltion into consideration that the image will be displayed onto.(i.e. the callees video screen)
```

Comment:
A definition of "too large" in terms of the background image size requirement would be very useful here.